lib/ObjWriter/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ endif()
 add_library(objwriter
    SHARED
    objwriter.cpp
+   objwriter.h
```

Comment:
ObjectWriter is only used by CoreRT's managed code P/Invoking to exported functions; there's no C++ code that needs to include it in compilation so no header is strictly necessary.